misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/config/ClientConfigKind.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ballerinalang.debugadapter.config;
+
+/**
+ * Defines the possible types of client(user) configurations.
+ *
+ * @since 2.0.0
+ */
+public enum ClientConfigKind {
+    ATTACH_CONFIG,
+    LAUNCH_CONFIG
+}
```

Comment:
Do we need a separate class here only for `ClientConfigKind`?

Reply:
Moved to `ClientConfigHolder`